tests/Cache/FileHandlerTest.php

```diff
@@ -138,7 +138,10 @@ public function testWriteWritesCache()
         $handler->write($cache);
 
         $this->assertFileExists($file);
-        $this->assertSame($cache->toJson(), file_get_contents($file));
```

Comment:
It's a bug that the tool doesn't pick it up when you assign to a variable first, since that assignment is live when we reach the assertion.